hack/version_test_ge_xyz.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+IFS=. read -r x y z <<-a
+$1
+a
+IFS=. read -r xx yy zz <<-a
+$2
+a
+printf "version: %s.%s.%s\nminimum: %s.%s.%s\n" "$x" "$y" "$z" "$xx" "$yy" "$zz" >&2
+test "$x" -gt "$xx" ||\
+{ test "$x" -eq "$xx" &&\
+        { test "$y" -gt "$yy" ||\
+        { test "$y" -eq "$yy" &&\
+                 test "$z" -ge "$zz";};};}
+set -- $?
+unset -v x y z xx yy zz
+exit "$1"
```

Comment:
This is cryptic and does not handle all possible cases.
We can greatly simplify this using python `packaging`[1] package.
It supports the common version numbers styles[2].

```python
#!/usr/bin/env python3
import sys
from packaging import version

if len(sys.argv) != 3:
    sys.exit("error: usage: check-version ACTUAL MINIMUM")

actual = version.parse(sys.argv[1])
minimum = version.parse(sys.argv[2])

if actual < minimum:
    sys.exit(f"error: version {actual} < minimum {minimum}")

print(f"version: {actual}")
```
Failing check:

```shell
$ hack/check-version 0.7.0 0.7.2; echo $?
error: version 0.7.0 < minimum 0.7.2
1
```

Successful check:

```shell
$ hack/check-version 0.8.0 0.7.2; echo $?
version: 0.8.0
0
```

[1] https://packaging.pypa.io/en/stable/version.html
[2] https://peps.python.org/pep-0440/